cristin-import/src/main/java/no/unit/nva/cristin/MappingConstants.java

```diff
@@ -0,0 +1,11 @@
+package no.unit.nva.cristin;
+
+public final class MappingConstants {
+
+    public static String MAIN_CATEGORY_BOOK = "BOK";
```

Comment:
Do you have a source for this? Maybe in documentation for Cristin?

Reply:
I am in contact with Daniel to find the source that contains this information